lib/manageiq/appliance_console/utilities.rb

```diff
@@ -16,6 +16,17 @@ def self.rake_run(task, params)
       result
     end
 
+    def self.run_rake!(task, params)
```

Comment:
Method name is `run_rake!` but callers use `ManageIQ::ApplianceConsole::Utilities.rake_run!`